### ISSUE TITLE
Cannot upgrade `camunda-bpmn-js` via `npm`

### ISSUE THREAD

**Issue by boris-petrov** [camunda/camunda-bpmn-js#145]
__Describe the Bug__

Running `npm update --save` after I've updated `camunda-bpmn-js` to 0.15.2 leads to:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: project@0.0.0
npm ERR! Found: @bpmn-io/properties-panel@0.15.0
npm ERR! node_modules/@bpmn-io/properties-panel
npm ERR!   peer @bpmn-io/properties-panel@"0.15.x" from camunda-bpmn-js@0.15.2
npm ERR!   node_modules/camunda-bpmn-js
npm ERR!     dev camunda-bpmn-js@"0.15.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @bpmn-io/properties-panel@"0.16.x" from bpmn-js-properties-panel@1.3.0
npm ERR! node_modules/bpmn-js-properties-panel
npm ERR!   peer bpmn-js-properties-panel@"1.x" from camunda-bpmn-js@0.15.2
npm ERR!   node_modules/camunda-bpmn-js
npm ERR!     dev camunda-bpmn-js@"0.15.2" from the root project
```

Actually leaving it to version 0.15.1 and running the same command again leads to an error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: project@0.0.0
npm ERR! Found: @bpmn-io/properties-panel@0.13.2
npm ERR! node_modules/@bpmn-io/properties-panel
npm ERR!   peer @bpmn-io/properties-panel@"0.13.x" from camunda-bpmn-js@0.15.1
npm ERR!   node_modules/camunda-bpmn-js
npm ERR!     dev camunda-bpmn-js@"0.15.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @bpmn-io/properties-panel@"0.16.x" from bpmn-js-properties-panel@1.3.0
npm ERR! node_modules/bpmn-js-properties-panel
npm ERR!   peer bpmn-js-properties-panel@"1.x" from camunda-bpmn-js@0.15.1
npm ERR!   node_modules/camunda-bpmn-js
npm ERR!     dev camunda-bpmn-js@"0.15.1" from the root project
```

I have to pin `bpmn-js-properties-panel` to `1.2.0` to overcome this issue. But then I can't update `camunda-bpmn-js`. It's a mess. :smile: 

__Steps to Reproduce__

Create a new empty directory, `cd` into it and run `npm install camunda-bpmn-js`. It fails.

__Expected Behavior__

To not have versioning conflicts like that.

__Environment__

- OS: Linux
- Modeler version: 0.15.2

**Comment by pinussilvestrus** [comment]
Hi @boris-petrov, thanks for reporting! Can you share the node / npm version you use?

**Comment by pinussilvestrus** [comment]
Thanks! Can you also share your `package.json`?

npm@8 is really strict when it comes to peer dependencies, which makes automated upgrades via `npm update` pretty hard.

I can reproduce that the `@bpmn-io/properties-panel` peer-dependencies of `camunda-bpmn-js@0.15.2` and `bpmn-js-properties-panel@0.12.0` are not matching. This needs an update.

**Comment by pinussilvestrus** [comment]
It seems we simply forgot to update the peer dependencies in the bpmn-js-properties-panel (https://github.com/bpmn-io/bpmn-js-properties-panel/commit/3c56c47193c76e30cf87ae22910093b568847c20). This should be already fixed via https://github.com/bpmn-io/bpmn-js-properties-panel/commit/45f112005cba42b0c1e34e6a40d26441062d3bf2. 

What's pending: integrate latest versions of `bpmn-js-properties-panel` and `@bpmn-io/properties-panel` into `camunda-bpmn-js`. /cc @philippfromme @barmac

**Comment by philippfromme** [comment]
Fixed by [0.15.3](https://www.npmjs.com/package/camunda-bpmn-js/v/0.15.3). 🎉
